components/views/dropdown-menu-view.js

```diff
@@ -0,0 +1,262 @@
+define([
+        'backbone',
+        'jquery',
+        'underscore',
+        'text!../templates/dropdown.underscore'
+    ],
+    function( Backbone, $, _, DropdownTpl ) {
+        'use strict';
+
+        /**
+         *  Dropdown Menu View which is fully accessible
+         *
+         *  Initialize the view by passing in the following attributes
+         *  className: 'space separated string of classes for element',
+         *  model: with the following attributes (example values added)
+         *      main: {
+         *          text: 'username',
+         *          url: 'dashboard'
+         *      },
+         *      button_label: 'User options dropdown',
+         *      items: [
+         *          {
+         *              text: 'Dashboard',
+         *              url: 'dashboard'
+         *          }, {
+         *              text: 'Account',
+         *              url: 'account_settings'
+         *          }, {
+         *              text: 'Profile',
+         *              url: 'learner_profile'
+         *          }, {
+         *              text: 'Sign Out',
+         *              url: 'logout'
+         *          }
+         *      ]
+         *  parent: 'selecter for parent element that will be replaced with dropdown menu'
+         *
+         */
+        return Backbone.View.extend({
```

Comment:
Nit: I know you had the opposite comment for me in my PR, but in the edx-platform we bind our views to a variable which we return. The benefit of this is that it is easier to search for the class name, because then it finds the definition as well as the usages. We should agree on which style to use...